mixins/arrow-keys-mixin.js

```diff
@@ -0,0 +1,145 @@
+import { html } from 'lit-element/lit-element.js';
+
+const keyCodes = Object.freeze({
+	END: 35,
+	HOME: 36,
+	LEFT: 37,
+	UP: 38,
+	RIGHT: 39,
+	DOWN: 40,
+});
+
+export const ArrowKeysMixin = superclass => class extends superclass {
+	static get properties() {
+		return {
+			arrowKeysDirection: { type: String },
+			arrowKeysNoWrap: { type: Boolean }
+		};
+	}
+
+	constructor() {
+		super();
+		this.arrowKeysDirection = 'leftright';
+		this.arrowKeysNoWrap = false;
+	}
+
+	arrowKeysContainer(inner) {
+		return html`<div class="arrow-keys-container" @keyup="${this._handleArrowKeys}">
+			${inner}
+		</div>`;
+	}
+
+	async focusablesProvider() {
```

Comment:
I think this should be `arrowKeysFocusablesProvider` to be consistent with the rest of the API.